CRR/clients.js

```diff
@@ -27,7 +29,34 @@ function setupClients({
     secretKey,
     endpoint,
 }, log) {
-    const awsConfig = {
+    const awsv3Config = {
+        region: 'us-east-1',
+        credentials: {
+            accessKeyId: accessKey,
+            secretAccessKey: secretKey,
+        },
+        endpoint,
+        forcePathStyle: true,
+        requestHandler: new NodeHttpHandler({
+            httpAgent: new http.Agent({ keepAlive: true }),
+            httpsAgent: new https.Agent({ 
+                keepAlive: true,
+                rejectUnauthorized: false
```

Comment:
Why this one ?

Reply:
Could we add a comment to say, in v2 ssl was anyway disabled, and so here we start accepting https but there will be no security checks enforced? Indeed today, we do not have TLS between internal services, so it cannot work. But that might change with the arrival of talos/cilium (if confirmed), where we may start enforcing internal security.